docs/Plugins-Guide.md

```diff
@@ -339,7 +383,9 @@ fastify
 ```
 
 <a name="custom-errors"></a>
+
 ## Custom errors
```

Comment:
All headings and manual anchors should remain together. I'm going to stop leaving comments on them, but this is definitely not the last instance of this issue.